app/models/drive_backup.rb

```diff
@@ -13,6 +14,10 @@ def backup
     end
   end
 
+  def available?
+    config.service_account_json.present?
```

Comment:
service_account_json seems a little vague. Where is this naming from? if this is the key that allows access to a specific google account I think it should be like google_api_key or something?